src/goSuggest.ts

```diff
@@ -227,10 +227,10 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				});
 			});
 		});
-		return setGocodeProps;
-		// return Promise.all([setPkgsList, setGocodeProps]).then(() => {
-		// 	return;
-		// });
+		// return setGocodeProps;
+		return Promise.all([setPkgsList, setGocodeProps]).then(() => {
```

Comment:
So this would mean if there is any delay to due to `gopkgs` it will only affect the first auto-completion result. Right?

Reply:
Yes... and `goImportablePackages(useCache=true)` so it should always use cache. Fast, doesn't have wait.
First time invocation can result to empty packages. But "preload packages" on startup will avoid the empty packages.